src/map/atcommand.c

```diff
@@ -5277,18 +5277,43 @@ ACMD(follow)
 }
 
 /*==========================================
- * @dropall by [MouseJstr]
- * Drop all your possession on the ground
- *------------------------------------------*/
+* @dropall by [MouseJstr] and [Xantara]
```

Comment:
Sorry i did not notice this before, please put back the space before the comments.
Afterwards, it should be ready to merge :)

Reply:
done